src/community/attributes/attributes-qpc.adoc

```diff
@@ -2,10 +2,33 @@
 
 // Product attributes
 :ProductName: Quipucords
-:ProductShortName: Quipucords
+:ProductNameMedium: Quipucords
+:ProductNameShort: Quipucords
 :ProductNameStartSentence: The Quipucords tool
-:ProductInCommands: qpc
+:ProductInCommands: quipucords
+:ProductInCommandsShort: qpc
 
 // Red Hat product attributes
+:CompanyName: Red Hat
+
+:RHELName: Red Hat Enterprise Linux
+:RHELNameShort: RHEL
+
+:AMQName: Red Hat AMQ
+:AMQNameShort: RHAMQ
+:DecisionManagerName: Red Hat Decision Manager
+:DecisionManagerNameShort: Decision Manager
+:DecisionManagerOld: Red Hat JBoss BRMS
+:EAPName: Red Hat JBoss Enterprise Application Platform
+:EAPNameShort: JBoss EAP
+:FuseName: Red Hat Fuse
+:FuseNameShort: Red Hat Fuse
+:FuseNameOld: Red Hat JBoss Fuse
+
+:SatelliteServerName: Red Hat Satellite Server
+:SatelliteServerNameShort: Satellite Server
+:SatelliteNameBrandedShort: Red Hat Satellite
+:SatelliteNameVeryShort: Satellite
 
 // External product attributes
+:vCenterServerName: vCenter Server
```

Comment:
Looks like the basic linting now needs to be expanded to encompass missing results, there appears to be some issues around the the `install` docs being produced... recommend we take a look at `ProductShortName` and `ProductMediumName` vs what's in the attributes which is flipped...

  
![Screen Shot 2019-07-29 at 4 23 44 PM](https://user-images.githubusercontent.com/3761375/62079991-c4610880-b21d-11e9-87cb-730c80407780.png)

![Screen Shot 2019-07-29 at 4 23 54 PM](https://user-images.githubusercontent.com/3761375/62080000-c9be5300-b21d-11e9-86ea-c6a938e6b455.png)

Reply:
Corrected attribute calls in install master.adoc files and the assemblies and content topics (the dummied structure for the POC build on the install side). Changes also included a new general "about qpc/discovery" topic started today.